docs/INSTALL.md

```diff
@@ -0,0 +1,658 @@
+
+In case of merge conflicts refer to [Resolving Conflicts - ILIAS Development Guide](http://www.ilias.de/docu/goto.php?target=pg_15604).
+  
+Afterwards you MUST open the ILIAS Installation Wizard in your browser (e.g. http://yourservername.org/setup/setup.php) and check if your database needs updates or hotfixes.
+
+As a last step you should log in with a User using your custom skin. If everything works fine change back from Delos to your skin. If not refer to [Customizing ILIAS](#customizing-ilias) to modify your skin to match the new requirements.
+
+<a name="information-on-updates"></a>
+## Information on Updates
+
+To keep your ILIAS Installation secure and healthy it is important that you keep it up to date. To get informations about updates and security fixes you SHOULD consider to subscribe to the ILIAS Admin Mailing-List: http://lists.ilias.de/cgi-bin/mailman/listinfo/ilias-admins
+
+<a name="upgrading-dependencies"></a>
+# Upgrading Dependencies
+
+When you upgrade from rather old versions please make sure that the dependencies, like MySQL and PHP, are up to date. Below you will find the supported versions for each ILIAS release.
+
+<a name="php"></a>
+## PHP
+
+| ILIAS Version   | PHP Version                           |
+|-----------------|---------------------------------------|
+| 5.2.x           | 5.5.x - 5.6.x, 7.0.x                  |
+| 5.0.x - 5.1.x   | 5.3.x - 5.5.x                         |
+| 4.4.x           | 5.3.x - 5.5.x                         |
+| 4.3.x           | 5.2.6 - 5.4.x                         |
+| 4.2.x           | 5.2.6 - 5.3.x                         |
+| 4.0.x - 4.1.x   | 5.1.4 - 5.3.x                         |
+| 3.8.x - 3.10.x  | 5.1.4 - 5.2.x                         |
+
+<a name="mysql"></a>
+## MySQL
+
+| ILIAS Version   | MySQL Version                         |
+|-----------------|---------------------------------------|
+| 4.4.x - 5.2.x   | 5.0.x, 5.1.32 - 5.1.x, 5.5.x, 5.6.x   |
+| 4.2.x - 4.3.x   | 5.0.x, 5.1.32 - 5.1.x, 5.5.x          |
+| 4.0.x - 4.1.x   | 5.0.x, 5.1.32 - 5.1.x                 |
+| 3.10.x          | 4.1.x, 5.0.x, 5.1.32 - 5.1.x          |
+| 3.7.3 - 3.9.x   | 4.0.x - 5.0.x                         |
+
+<a name="imagemagick"></a>
+## ImageMagick
+
+| ILIAS Version   | ImageMagick Version                   |
+|-----------------|---------------------------------------|
+| 4.2.x - 5.2.x   | 6.3.8-3 or higher                     |
+| < 4.2.x         | No specific version requirements      |
+
+<a name="contribute"></a>
+# Contribute
+
+We have a big [community](http://www.ilias.de/docu/goto.php?target=cat_1444&client_id=docu) and you can get a member of [ILIAS Society](http://www.ilias.de/docu/goto.php?target=cat_1669&client_id=docu).
+You may even join us at one of our regular [ILIAS Conferences](http://www.ilias.de/docu/goto.php?target=cat_2255&client_id=docu).
+
+<a name="pull-requests"></a>
+## Pull Requests
```

Comment:
I'm happy to split this section into several parts or rename it to "Further reading/links" as I think it's a good thing to have the Feature Wiki and Bugtracker mentioned. Is /docs/CONTRIBUTING.md the leading file for informations on pull requests? IIRC I got this part from an (now archived) document on the ILIAS Wiki.